docs/reference/builder.md

```diff
@@ -1245,7 +1244,7 @@ The `WORKDIR` instruction sets the working directory for any `RUN`, `CMD`,
 If the `WORKDIR` doesn't exist, it will be created even if it's not used in any
 subsequent `Dockerfile` instruction.
 
-It can be used multiple times in the one `Dockerfile`. If a relative path
+It can be used multiple times in one `Dockerfile`. If a relative path
```

Comment:
We should probably not use `one` here, as it's a bit confusing. While you're at it, could you change "It" to something more meaningful? I suggest;

```Markdown
The `WORKDIR` instruction can be used multiple times in a `Dockerfile`. If a
relative path is provided, it will be relative to the path of the previous
`WORKDIR` instruction. For example:
```

There's still some room for improvement, but that can be addresses another time